risc0/zkvm/src/host/client/prove/mod.rs

```diff
@@ -120,6 +121,19 @@ pub struct ProverOpts {
     // moment if there is a better place. At some point before 1.0, this option should be moved or
     // dropped.
     pub prove_guest_errors: bool,
+    /// The kind of receipt to be generated.
+    pub receipt_format: ReceiptKind,
```

Comment:
```suggestion
    pub receipt_kind: ReceiptKind,
```

Reply:
thanks for catching this. I did a search/replace locally